src/state/symbols/reducer.js

```diff
@@ -31,6 +31,13 @@ export function symbolsReducer(state = initialState, action) {
           symbol = 'pBCH'
         }
 
+        if (id.includes('F0') && id !== 'USTF0') {
```

Comment:
Remember to include TESTUSDTF0 here also
['TESTUSDTF0', 'USTF0'].includes(symb)